index.xml

```diff
@@ -12005,12 +11827,51 @@ interface
                     Try running the following steps:
                     <ol>
                       <li>
+                        Let <var>O</var> be <span class='esvalue'>null</span>.
+                      </li>
+
+                      <li>
+                        If <var>target</var> is not a <a class='dfnref'
```

Comment:
It's not obvious at this point that `target` can only be a namespace or interface.  So I think it would be clearer to make this check "If `target` is an interface".  And perhaps assert at the top of the algorithm that it's one or the other.

But also, calling this thing `target` is a bit weird.  It's more like "source" or something....